docs/rules.rst

```diff
@@ -42,6 +45,8 @@ Note: 22X, 23X and 24X has not had asyncio-specific suggestions written.
 - **ASYNC250**: Builtin ``input()`` should not be called from async function. Wrap in ``[trio/anyio].to_thread.run_sync()`` or ``asyncio.loop.run_in_executor()``.
 - **ASYNC251**: ``time.sleep(...)`` should not be called from async function. Use ``[trio/anyio/asyncio].sleep(...)``.
 
+.. _async900:
```

Comment:
```suggestion
.. _async9xx:
```
to make it clear this is for the category

Reply:
I wanted to link to 900 in particular but couldn't get the reference to work... but trying it again now it works without issue so I guess I'll just move it there :upside_down_face: